### PR TITLE
Address all intellij warnings for Channel.scala

### PR DESCRIPTION
**Pull request by pm47**
Fixing those small warnings one by one in unrelated PRs makes reviews more difficult.

### CODE REVIEW

File: eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala [hunk 1]
```diff
@@ -47,33 +47,33 @@ import scala.util.{Failure, Success, Try}
...
-  val MIN_DUSTLIMIT = 546 sat
+  val MIN_DUSTLIMIT: Satoshi = 546 sat
 
   // we won't exchange more than this many signatures when negotiating the closing fee
   val MAX_NEGOTIATION_ITERATIONS = 20
 
-  // this is defined in BOLT 11
-  val MIN_CLTV_EXPIRY_DELTA = CltvExpiryDelta(18)
-  val MAX_CLTV_EXPIRY_DELTA = CltvExpiryDelta(7 * 144) // one week
+  // this is private defined in BOLT 11
```
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala:65]: What do you mean?

Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala]: It's just the result of a reckless replace to convert all `def` to `private def`...

Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala]: Fixed with 8b8b042.

**Review by t-bast — APPROVED** [review]

**Comment by pm47** [comment]
I think I can merge this, there shouldn't be many conflicts with #1702 WDYT @t-bast?

**Comment by t-bast** [comment]
Sure, let's merge this, if it does conflict it should be a simple rebase which shouldn't change anything in #1702

**Pull request merged**